libs/tools/barista/src/transform.ts

```diff
@@ -300,3 +342,17 @@ export function internalContentTransformerFactory(
     return transformed;
   };
 }
+
+// Checks whether a URL is relative
+function isRelativeUrl(href: string): boolean {
+  return !href.match(/^(?:[a-z]+:)?\/\//i);
```

Comment:
Please provide a URL to a sample regex101 or something similar in a comment where we have a reference for this regex.

for example:

```suggestion
  // Matches every character case insensitive until a double point is coming. Has to have at least one character.
  // https://regex101.com/r/quBotu/3/
  return !href.match(/^(?:[a-z]+:)?\/\//i);
```